hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -153,34 +153,75 @@ tests:
         exp_alerts: [ ]
 
 
+    # vmi running on a node with an unready virt-handler pod
+  - interval: 1m
+    input_series:
+      - series: 'kube_pod_info{pod="virt-launcher-testvm-123", node="node01"}'
+        values: '1 1 1 1 1 1 1 1 1 1 1'
+      - series: 'kube_pod_status_ready{pod="virt-handler-asdf", condition="true"}'
+        values: '_ _ _ _ _ _ _ _ _ _ _'
```

Comment:
i think that it should be something like :
```
      - series: 'kube_pod_status_ready{pod="virt-handler-asdf", condition="false"}'
        values: '1 1 1 1 1 1 1 1 1 1 1'
```

Reply:
Ok , Ill add it